Specs/Sequence/Chinese/URLModel.json

```diff
@@ -1,449 +1,4 @@
 [
-  {
-    "Input": "https://abc.com",
```

Comment:
Please keep all cases as the single URLs should still work in Chinese.
Also, the other cases should be translated to Chinese, ok?